apps/service-portal/infra/service-portal.ts

```diff
@@ -5,6 +5,15 @@ export const serviceSetup = (services: {}): ServiceBuilder<'service-portal'> =>
     .namespace('service-portal')
     .liveness('/liveness')
     .readiness('/readiness')
+    .replicaCount({
+      default: 2,
+      max: 20,
+      min: 2,
+    })
+    .resources({
+      limits: { cpu: '400m', memory: '512Mi' },
+      requests: { cpu: '200m', memory: '256Mi' },
+    })
```

Comment:
Not sure why this is needed but maybe it's not a bad idea.

Reply:
Neither was present, so I figured I'd add it, since I wanted to double resources there as well, away from the default.